File: src/main/kotlin/br/com/fiap/restfullspringapi/RestfullSpringApiApplication.java
package br.com.fiap.restfullspringapi;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.stereotype.Controller;

@SpringBootApplication
@EnableAutoConfiguration
@Controller
public class RestfullSpringApiApplication {
    public static void main(String[] args) {
        SpringApplication.run(RestfullSpringApiApplication.class, args);
    }
}

File: src/main/kotlin/br/com/fiap/restfullspringapi/RestfullSpringApiApplication.kt
package br.com.fiap.restfullspringapi

import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.ResponseBody


@SpringBootApplication
@EnableAutoConfiguration
@Controller
class RestfullSpringApiApplication {
    companion object {

    }


    @RequestMapping("/")
    @ResponseBody
    fun home(): String? {
        return "Hello World!"
    }
}
fun main(args: Array<String>) {
    runApplication<RestfullSpringApiApplication>(*args)
}
